mlflow/azure/client.py

```diff
@@ -0,0 +1,81 @@
+from copy import deepcopy
+from six.moves import urllib
```

Comment:
```suggestion
import urllib
```

Should we fix the import statement here since we've excluded `six` from MLflow's dependencies?